file_paths.md

```diff
@@ -253,6 +254,7 @@ create a `puppet` user or group.
                 pxp-access.log                # not enabled by default
             spool *                           # directory containing results of pxp-agent modules
             run *
+        tasks-cache *                         # directory containing a cache of files for all downloaded tasks
```

Comment:
I'm wondering if this should be a sibling directory to `spool` under `var`, so that's closer to the *nix layout?